src/ttauri/text/true_type_font.hpp

```diff
@@ -92,7 +92,7 @@ class true_type_font final: public font {
             parsefontDirectory();
 
         } catch (std::exception const &e) {
-            throw parse_error("{}: Could not parse font directory.\n{}", url, e.what());
+            throw parse_error("{}: Could not parse font directory.\n{}", url.directory(), e.what());
```

Comment:
Did this give an error before?
It should be possible format an URL. Also I wonder if directory() would drop of the last element.

If you can't get automatic formatting to work, there is a to_string(URL const &) function that you can use here instead, so I can merge this.